controls/core.sp

```diff
@@ -0,0 +1,397 @@
+        oci_core_instance_metric_cpu_utilization_daily
+      where
+        date_part('day', now() - timestamp) <=30
+      group by
+        id
+    )
+    select
+      i.id as resource,
+      case
+        when avg_max is null then 'error'
+        when avg_max < 20 then 'alarm'
+        when avg_max < 35 then 'info'
+        else 'ok'
+      end as status,
+      case
+        when avg_max is null then 'Metrics not available for ' || i.title || '.'
+        else i.title || ' averaging ' || avg_max || '% max utilization over the last ' || days || ' days.'
+      end as reason,
+      i.region,
+      coalesce(c.name, 'root') as compartment
+    from
+      oci_core_instance as i
+      left join core_instance_utilization as u on u.id = i.id
+      left join oci_identity_compartment as c on c.id = i.compartment_id;
+  EOT
+
+  tags = merge(local.core_common_tags, {
+    class = "unused"
+  })
+}
+
+control "core_public_ip_unattached" {
+  title       = "Unused reserved public IP addresses should be removed"
+  description = "Unattached reserved public IP addresses cost money and should be released."
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      a.id as resource,
+      case
+        when a.lifecycle_state = 'AVAILABLE'  then 'alarm'
+        else 'ok'
+      end as status,
+      a.display_name || ' in ' || a.lifecycle_state || ' state.' as reason,
+      coalesce(c.name, 'root') as compartment
+    from
+      oci_core_public_ip as a
+      left join oci_identity_compartment as c on c.id = a.compartment_id;
+  EOT
+
+  tags = merge(local.core_common_tags, {
+    class = "unused"
+  })
+}
+
+control "core_volume_auto_tune_performance_enabled" {
+  title       = "Block volume should be enabled with auto-tune for better performance"
+  description = "Block volume auto-tune should be enabled for better performance."
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      a.id as resource,
+      case
+        when is_auto_tune_enabled then 'ok'
+        else 'alarm'
+      end as status,
+      case
+        when is_auto_tune_enabled then a.title || ' auto-tune volume performance enabled.'
+        else a.title || ' auto-tune volume performance disabled.'
+      end as reason,
+      a.region,
+      coalesce(c.name, 'root') as compartment
+    from
+      oci_core_volume as a
+      left join oci_identity_compartment as c on c.id = a.compartment_id
+  EOT
+
+  tags = merge(local.core_common_tags, {
+    class = "deprecated"
+  })
+}
+
+control "core_volume_backup_age_90" {
+  title       = "Volume backup created over 90 days ago should be deleted if not required"
+  description = "Old backups are likely unneeded and costly to maintain."
+  severity    = "low"
+
+  sql = <<-EOT
+    select
+      a.id as resource,
+      case
+        when a.time_created > current_timestamp - interval '90 days' then 'ok'
+        else 'alarm'
+      end as status,
+      a.display_name || ' created at ' || a.time_created || '.' as reason,
```

Comment:
For this reason, and others in this benchmark (or others), can we also include the `(num_days)` as well?